config/crds/patches/apm-kibana-patches.yaml

```diff
@@ -21,3 +21,13 @@
 # that would maybe not match the user's k8s version.
 - op: remove
   path: /spec/validation/openAPIV3Schema/properties/spec/properties/podTemplate/properties
+
+# without this the API server complains with "The CustomResourceDefinition "elasticsearches.elasticsearch.k8s.elastic.co" is invalid: spec.validation.openAPIV3Schema.type: Required value: must not be empty at the root" on 1.15 at least. When we stop supporting 1.11 we should remove these as well.
```

Comment:
We should include something like
```
# TODO: remove once https://github.com/kubernetes-sigs/controller-tools/issues/392 is fixed
```
IMO this is a bug in controller-tools.
Also it looks like this errors out in 1.14 as well, so we probably want to remove the 1.11 mention.

Reply:
Nit: can you reformat the comments to multiple lines so they read better on a small-ish screen?